src/EnergyPlus/Humidifiers.hh

```diff
@@ -82,6 +82,8 @@ namespace Humidifiers {
         Variable,
         Num
     };
+    static constexpr std::array<std::string_view, static_cast<int>(InletWaterTemp::Num)> inletWaterTempsUC = {"FixedInletWaterTemperature",
+                                                                                                              "VariableInletWaterTemperature"};
```

Comment:
inletWaterTempUC? doesn't look UC.

Reply:
> inletWaterTempsUC? doesn't look UC.

Was I supposed to actually fix them or something?  


















Thanks @rraustad :)